src/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-bottom: 2rem;
+
+  animation: slide-in 0.2s ease-in;
+
+  border-radius: 12px;
+
+  background: rgba(37, 37, 37, 0.6);
+
+  color: #fff;
```

Comment:
I'm not sure what is this color doing since we already have the color on the content and the title, is there a reason? 💭

Reply:
I must have overlooked it thanks 🙏

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28